### ISSUE TITLE
BundleImportExport is missing `selection_can_change_qty`

### ISSUE THREAD

**Issue by barryvdh** [magento/magento2#9342]
### Preconditions
1. Magento 2.1.6

### Steps to reproduce
1. Create a Bundle product with options.
2. Export the product, no information about `selection_can_change_qty`
3. Import, no possibility to set `selection_can_change_qty`

![image](https://cloud.githubusercontent.com/assets/973269/25279627/5bd982f0-26a7-11e7-98b1-0d368fa19bae.png)

### Expected result
`selection_can_change_qty` is exported and imported correctly

### Actual result
`selection_can_change_qty` is always `true`/`1`

### Suggested solution
It seems that while importing, the value is hardcoded: https://github.com/magento/magento2/blob/2.1.6/app/code/Magento/BundleImportExport/Model/Import/Product/Type/Bundle.php#L310

I suggest a parameter `can_change_qty` will be added, while keeping the default on `1` to preserve BC.

```
'selection_can_change_qty' => (isset($selection['can_change_qty'])) ? ($selection['can_change_qty'] ? 1 : 0) : 1,
```

The value should also be added to the export of a bundle, to keep import <-> export in sync.

**Comment by IlnitskiyArtem** [comment]
@barryvdh, Thanks for reporting this issue. We've created internal ticket MAGETWO-70390 to address this issue.

**Comment by magento-engcom-team** [comment]
@barryvdh, thank you for your report.
We've created internal ticket(s) MAGETWO-70390 to track progress on the issue.

**Comment by piotrekkaminski** [comment]
This issue was moved to magento-engcom/import-export-improvements#41

**Comment by magento-engcom-team** [comment]
Hi @barryvdh. Thank you for your report. 
The issue has been fixed in magento-engcom/import-export-improvements#100 by @adam-paterson in 2.3-develop branch
Related commit(s):
 - [77fefd8a2292c1011b5bac9d4851590703825b95](https://github.com/magento/magento2/commit/77fefd8a2292c1011b5bac9d4851590703825b95)
 - [fe9054e71661a8a4a52a70f3aa6dc4a9dd5e7d89](https://github.com/magento/magento2/commit/fe9054e71661a8a4a52a70f3aa6dc4a9dd5e7d89)

The fix will be available with the upcoming 2.3.0 release.
